modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/json_serializable/class.mustache

```diff
@@ -0,0 +1,83 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part '{{classFilename}}.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  createToJson: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class {{{classname}}} {
+{{>serialization/json_serializable/dart_constructor}}
+
+{{#vars}}
+  {{#description}}
+      /// {{{description}}}
+  {{/description}}
+  {{^isEnum}}
+      {{#minimum}}
+          // minimum: {{{minimum}}}
+      {{/minimum}}
+      {{#maximum}}
+          // maximum: {{{maximum}}}
+      {{/maximum}}
+  {{/isEnum}}
+  {{^isBinary}}
+  @JsonKey(
+    {{#defaultValue}}defaultValue: {{{defaultValue}}},{{/defaultValue}}
+    name: r'{{{baseName}}}',
+    required: {{#required}}true{{/required}}{{^required}}false{{/required}},
```

Comment:
needs `includeIfNull` to be used in conjunction with `required`. If field is required, then `null` value must be included

Reply:
I believe I have addressed this now